circuits-circom/venmo_receive.circom

```diff
@@ -88,13 +97,10 @@ template VenmoReceiveEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
     order_id_squared <== order_id * order_id;
 }
 
-// In circom, all output signals of the main component are public (and cannot be made private), the input signals of the main component are private if not stated otherwise using the keyword public as above. The rest of signals are all private and cannot be made public.
-// This makes modulus, order_id and signature public. Signature being public means it allows the mailserver to trace who the offender is.
-
 // Args:
 // * max_header_bytes = 1024 is the max number of bytes in the header
 // * max_body_bytes = 6400 is the max number of bytes in the body after precomputed slice
 // * n = 121 is the number of bits in each chunk of the modulus (RSA parameter)
 // * k = 17 is the number of chunks in the modulus (RSA parameter)
 // * pack_size = 7 is the number of bytes that can fit into a 255ish bit signal (can increase later)
-component main { public [ modulus, signature, order_id ] } = VenmoReceiveEmail(1024, 6400, 121, 17, 7);
+component main { public [ order_id ] } = VenmoReceiveEmail(1024, 6400, 121, 17, 7);
```

Comment:
Assuming we're now verifying the modulus_hash vs modulus array in the smart contracts

https://github.com/zkemail/zk-email-verify/blob/main/packages/circuits/email-verifier.circom#L108-L109

Reply:
Yup.